try.html

```diff
@@ -538,6 +538,10 @@ <h3 id="version"> Version </h3>
   <td><img src='/cpan/v/Config-Augeas.svg' alt=''/></td>
   <td><code>https://img.shields.io/cpan/v/Config-Augeas.svg</code></td>
   </tr>
+  <tr><th data-keywords='R'> CRAN: </th>
+  <td><img src='/cran/v/devtools.svg' alt=''/></td>
+  <td><code>https://img.shields.io/cran/v/devtools.svg</code></td>
+  </tr>
```

Comment:
The example badge is rendering like this.

<img width="91" alt="screen shot 2017-03-29 at 10 39 53 pm" src="https://cloud.githubusercontent.com/assets/1487036/24485205/bea97130-14d0-11e7-996c-c48ca8c0b041.png">

Could you fix that?

Could you also add an entry for `/cran/l`?